harness/tests/bootstrap.rs

```diff
@@ -18,10 +18,6 @@ pub async fn test_static_bootstrap() {
 
     sleep(Duration::from_secs(1)).await;
 
-    let raft_1 = rafts.get(&1).unwrap();
-
-    wait_for_until_cluster_size_increase(raft_1.clone(), 3).await;
```

Comment:
Why is it necessary to remove `wait_for_until_cluster_size_increase`?
This test is meant to ensure that nodes 1, 2, and 3 are all functioning properly.
(This has been replaced with waiting until the cluster size reaches 3 to confirm that all three nodes are functioning properly.)

So, if `wait_for_until_cluster_size_increase` is removed, this test code will lose its meaning.

Reply:
I think the wait_until_rafts_ready function would ensure the cluster size increases